.github/workflows/ci.yml

```diff
@@ -0,0 +1,56 @@
+name: PHP Composer
+
+on:
+  push:
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+
+    strategy:
+      fail-fast: false
+      matrix:
+        php: [ '7.2', '7.3', '7.4' ]
```

Comment:
Could we update the matrix here to support also newer versions?

Reply:
@dereuromark The [propel/propel1](https://github.com/propelorm/Propel) library is not compatible with PHP 8. Since it seemed like the library isn't being maintained anymore, we forked both Propel and PropelBundle to add compatibility for PHP 8. I published 1.8.x versions of both packages yesterday.

https://packagist.org/packages/dayspring-tech/propel-bundle#1.8.0
https://packagist.org/packages/dayspring-tech/propel1#1.8.1